docs/solutions/sql_server_always_on_availability_groups/env-vars

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# OCI Account authentication information
+export TF_VAR_tenancy_ocid=<Tenancy OCID>
+export TF_VAR_user_ocid=<OCI User OCID that will deploy SQL AO>
+export TF_VAR_region=<Region to deploy SQL AO>
+export TF_VAR_private_key_path=<Private portion of API key associated with user>
+export TF_VAR_fingerprint=<Fingerprint of public portion of API key>
+
```

Comment:
add: export TF_VAR_private_key_password=""

Reply:
Please add this for release.

Reply:
Done